projects/libs/core-library-typescript/src/consensus/block-builder.ts

```diff
@@ -0,0 +1,36 @@
+import { types } from "../common-library";
+
+export default class BlockBuilder {
+    private virtualMachine: types.VirtualMachineClient;
```

Comment:
You are accidentally using 4 spaces in this file as well. If you'd configure vscode to use `.editorconfig`, it'll fix this automatically for you.